lms/djangoapps/discussion/templates/discussion/edx_ace/responsenotification/email/body.html

```diff
@@ -2,33 +2,29 @@
 
 {% load i18n %}
 {% load static %}
-
-{% block preview_text %}
-    {% blocktrans trimmed %}
-        {{ comment_username }} replied to your thread.
-    {% endblocktrans %}
-{% endblock %}
-
 {% block content %}
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
     <tr>
         <td>
-            {% blocktrans trimmed %}
-                <p>
-                    Hi {{ thread_username }},
-                </p>
+            <p>
+                {% blocktrans trimmed %}
+                    {{ comment_username }} replied to your <b>{{ thread_title }}</b> post
+                {% endblocktrans %}
+            </p>
```

Comment:
Does the endblocktrans need to be on L28? Or is here ok?

Reply:
Here is ok. I didn't want to include a lot of HTML in the blocktrans since translators will have to deal with interpolating the HTML with the text.